doc/man3/BIO_ctrl.pod

```diff
@@ -167,7 +167,7 @@ the case of BIO_seek() on a file BIO for a successful operation.
 =head1 HISTORY
 
 The BIO_get_ktls_send() and BIO_get_ktls_recv() macros were added in
-OpenSSL 3.0.
+OpenSSL 3.0. They were modified to never return -1 in OpenSSL 3.0.3.
```

Comment:
3.0.3 in already out without this, so I think it will be 3.0.4

Reply:
Good point! Thanks for the heads up. I'll update the PR.